ginivision/src/main/java/net/gini/android/vision/review/ReviewFragmentImpl.java

```diff
@@ -425,27 +433,24 @@ public void onFailed() {
 
     private void proceedToAnalysisScreen() {
         LOG.info("Proceed to Analysis Screen");
-        mListener.onProceedToAnalysisScreen(DocumentFactory.newDocumentFromPhoto(mPhoto));
+        mListener.onProceedToAnalysisScreen(
+                DocumentFactory.newDocumentFromPhotoAndDocument(mPhoto, mDocument));
     }
 
     private void applyRotationToPhoto(@NonNull PhotoEdit.PhotoEditCallback callback) {
         if (mPhoto == null) {
             return;
         }
         LOG.debug("Rotating the Photo {} degrees", mCurrentRotation);
-        mPhoto.edit()
```

Comment:
As mentioned above, for me it is more readable this that having all in one line.